src/picker/styles/variables.scss

```diff
@@ -14,6 +14,7 @@
   --num-columns: 8; /* How many columns to display in the emoji grid */
   --outline-size: 2px; /* Focus outline width */
   --border-size: 1px; /* Width of border used in most of the picker */
+  --picker-border-radius: 0.5rem; /* Border radius of the entire picker */
```

Comment:
I'd prefer to set this to `0` or `initial` instead. Changing the default styles is kinda-sorta a breaking change (maybe some people want no border radius!).